packages/core/src/delegation/DelegationNode.utils.ts

```diff
@@ -71,8 +72,25 @@ export async function countNodeDepth(
   return delegationTreeTraversalSteps
 }
 
+/**
+ * [ASYNC] Syncronise a delegation node with the latest state as stored on the blockchain.
+ *
+ * @param delegationNode The input delegation node for which the state needs to be syncronised.
+ *
+ * @returns A new instance of the same [DelegationNode] containing the up-to-date state fetched from the chain.
+ */
+export async function getSyncedState(
+  delegationNode: DelegationNode
```

Comment:
```suggestion
  delegationNode: IDelegationNode
```

Reply:
We don't have this at all anymore. Resolving it then.